src/backends/github/API.js

```diff
@@ -322,6 +322,7 @@ export default class API {
   }
 
   editorialWorkflowGit(fileTree, entry, filesList, options) {
+    entry.slug = options.collectionName ? `${ options.collectionName }-${ entry.slug }` : entry.slug;
```

Comment:
here i change the slug to include the collection name,
the slug is changed only before publishing and will be back to the normal name after publishing (since the issue is when a draft is saved and not when published), and this allows creating a PR with a better name

Reply:
Just transform `contentKey`, not the slug itself. It may not hurt anything now, but it could down the road when a developer doesn't notice we're changing the slug here (hence our use of constants practically everywhere).